mne/report/js_and_css/report.js

```diff
@@ -7,6 +7,12 @@ const refreshScrollSpy = () =>{
   })
 }
 
+const propagateScrollSpyURL = () =>{
+  $(window).on('activate.bs.scrollspy', function (event) {
+    history.replaceState({}, "", event.relatedTarget);
+  });
+}
```

Comment:
@hoechenberger looks like this works :+1: 

Note I intentionally used `replaceState` here because scrolling shouldn't affect history and the forward/back buttons, whereas clicking the links on the left (which set `window.location.href`) should

Reply:
great!